nginx/default.conf

```diff
@@ -0,0 +1,38 @@
+server {
```

Comment:
Please be hard on this, as I reused what I found

Is this good? Is nginx fine?

Reply:
I don't have enough expertise to review this part. But @sandhose or @vhf probably do.

Reply:
there could be a way to make nginx work as non-root, but it's not a priority for now. Otherwise LGTM